tests/bug01665-0.phpt

```diff
@@ -0,0 +1,21 @@
+--TEST--
```

Comment:
← I usually start counting at `1` and not `0` for the test files. Can you change the filename? For new files, I also usually would use `001` and no longer `1`.

Reply:
Alright, I'll then change the file order so that the 00X matches the xdebug.collect_params=X value (hence only the xdebug.collect_params=0 won't match and will be 006)